packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -285,9 +290,57 @@ class VmServiceWrapper implements VmService {
         },
       });
     }
+
+    await _addProcessedUrlsToTraceObject(
+        isolateId, resolvedUrlMap, traceObject);
+
     return CpuProfileData.parse(traceObject);
   }
 
+  // Helper function for fetching simplified package urls for each of
```

Comment:
nit: use /// for dartdoc. https://dart.dev/guides/language/effective-dart/documentation#do-use--doc-comments-to-document-members-and-types

And the first sentence should be followed by a single blank line.
e.g.
```
/// This is my method that does something.
///
/// more details. blah blah blah
```

Reply:
so I left these as just double because I wanted to make sure that my description seemed sufficient before promoting it to ///. I can definitely do that right off the bat though :)